README.md

```diff
@@ -1,3 +1,48 @@
 # atom-patch [![Build Status](https://travis-ci.org/atom/atom-patch.svg?branch=master)](https://travis-ci.org/atom/atom-patch)
 
 This data structure efficiently represents a transformation from input to output text, and it's useful for aggregating and combining changes that occur at different points in time and space.
+
+## Contributing
+
+```bash
```

Comment:
Is this stuff required if the contributor has no interest in recompiling the flatbuffers schema? If not, it might make sense to make that clear.